sample-cf-install-values.yml

```diff
@@ -5,6 +5,9 @@ app_domains:
 #@overlay/append
 - apps.cf.example.com
 
+#! set this if you want to use a reserved static IP for istio's load balancer.
```

Comment:
dont think we should add this to sample, as i think we were trying to keep sample-cf-install-values.yml to only inlcude required values.